### PR TITLE
Make the docker image more usable

### PR DESCRIPTION
**Pull request by bartv2**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | yes/no
| Deprecations? | yes/no
| Tests pass?   | yes/no
| Documented?   | yes/no
| Fixed tickets | #... 
| License       | MIT

### CODE REVIEW

**Comment by mathroc** [comment]
composer 2 also supports parallel fetching of the dependencies, maybe it’ll be easier for maintenance to just wait for a stable release of composer (or include the alpha version)

File: .dockerignore [hunk 1]
```diff
@@ -5,6 +5,6 @@
 !/benchmarks/
 !/composer.json
 !/phpunit.xml.*
-!/phpstan.neon.*
+!/phpstan*.neon
```
Comment (mcg-web) [inline .dockerignore:8]: why this change?

File: Dockerfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ FROM scratch AS test_source
 COPY benchmarks/ benchmarks/
 COPY src/ /src/
 COPY tests/ /tests/
-COPY phpunit.xml.* phpstan.neon.* .php_cs.* phpbench.json /
+COPY phpunit.xml.* phpstan*.neon .php_cs.* phpbench.json /
```
Comment (mcg-web) [inline Dockerfile:10]: why this change?

File: .dockerignore (hunk 1 above)
Comment (murtukov) [reply .dockerignore:8]: @mcg-web I think because i renamed the `phpstan.neon.dist` long time ago

Comment (bartv2) [reply .dockerignore:8]: Yes the name changed and the config wasn't included

**Comment by bartv2** [comment]
composer install is slow, doing parallel downloads helps a lot. This was the solution that i also use in other projects.

File: .dockerignore (hunk 1 above)
Comment (mcg-web) [reply .dockerignore:8]: ok nice, thank you!
